hm_cfg_files/config/CFG/radioss2020/MAT/LAW95.cfg

```diff
@@ -44,6 +44,8 @@ ATTRIBUTES(COMMON)
     SUBGRP_HEAT_MAT         = VALUE(SUBOBJECT, "");
     THERM_STRESS            = VALUE(INT, "Therm Stress");
     SUBGRP_THERM_STRESS     = VALUE(SUBOBJECT, "");
+    VISC_PRONY_option                       = VALUE(INT,"Visc Prony");
```

Comment:
maybe some incorrect indentation here ?

Reply:
Hi @sebastienVilleneuve ,

You are right. Please reject this pull request, and I'll do it again, after correction.
Thank you